packages/node_modules/@webex/plugin-meetings/src/meeting-info/request.js

```diff
@@ -24,9 +24,8 @@ export default class MeetingInfoRequest extends StatelessWebexPlugin {
     if (!options || !options.type || !options.destination) {
       throw new ParameterError('MeetingInfo should be fetched with a type and destination specified, see list of valid types and their corresponding values in constants');
     }
-    const format = MeetingInfoUtil.getMeetingInfoAPIFormat(options);
-    const resourceUrl = MeetingInfoUtil.getResourceUrl(format.type, format.value);
-    const requestParams = MeetingInfoUtil.getRequestParams(resourceUrl, format.type, format.value, API.LOCUS);
+    const resourceUrl = MeetingInfoUtil.getResourceUrl(options.type, options.destination);
+    const requestParams = MeetingInfoUtil.getRequestParams(resourceUrl, options.type, options.destination, API.LOCUS);
```

Comment:
For readability, could this be refactored to

```js
const requestParams = MeetingInfoUtil.getRequestParams(
  resourceUrl,
  options.type,
  options.destination,
  API.LOCUS
);
```

Reply:
will do it in the next PR . I don't want to re run the test